fitbenchmarking/results_processing/base_table.py

```diff
@@ -328,10 +329,32 @@ def create_pandas_data_frame(self, html=False):
                 elif len(minimizers_list[i][1]) < len(formatted[1]):
                     minimizers_list[i] = formatted
 
-        columns = pd.MultiIndex.from_tuples(minimizers_list)
-        table = pd.DataFrame.from_dict(str_results,
-                                       orient='index',
-                                       columns=columns)
+        multi_columns = pd.MultiIndex.from_tuples(minimizers_list)
+
+        # Store info on problem size
+        n_data_points = [results[0].get_n_data_points()
+                         for results in self.sorted_results.values()]
+        print(n_data_points)
+        n_params = [results[0].get_n_parameters()
+                    for results in self.sorted_results.values()]
+
+        # Build strings for problem size to display in table
+        self.problem_sizes = [str(m) + ' params, ' + str(n) + ' points'
+                              for m, n in zip(n_params, n_data_points)]
+
+        single_index = list(str_results.keys())
+
+        multi_index = pd.MultiIndex.from_tuples(zip(single_index,
+                                                    self.problem_sizes))
+
+        results_only = np.array(list(str_results.values()))
+        reshaped_results = results_only.reshape((len(multi_index),
+                                                 len(multi_columns)))
```

Comment:
I haven't used numpy arrays in a while but I'd have expected them to maintain the right shape. Did you check that it's not just that the array needs transposing (`results_only.T`)?

Reply:
Just checked this, it looks like reshaping (lines 351 and 252) is not necessary, as I get the same table if I build it directly with the array "result_only". So shall we just remove the reshaping and use "result_only" for building the table?

Reply:
Also, I tried using results_only.T but that raises an error saying "Shape of passed values is (6, 16), indices imply (16, 6)"